scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/CheckpointSimulationFacade.java

```diff
@@ -89,15 +88,6 @@ public CheckpointSimulationFacade(
     );
   }
 
-  /**
-   * Returns the total simulated time
-   * @return
-   */
-  @Override
-  public Duration totalSimulationTime(){
-    return totalSimulationTime;
-  }
-
```

Comment:
Can you give an example of what such a test would look like? While I see the hypothetical value in having a metric that can assess that we're selecting checkpoints that minimize the amount of time to be simulated, I cannot picture what a test using it would actually look like.

Reply:
We could add one line to every scheduler test that would assert the current "simulation time".

Reply:
Method has been restored.